File: program_data/github_java_sort_function/6/63.java
class RadixSort {
	public static int getMax(int[] nums) {
		int max = 0;
		for (int num : nums) {
			max = Math.max(max, num);
		}
		return max;
	}

	
	public static void countSort(int[] nums, int exponent) {
		int n = nums.length;
		int[] output = new int[n];
		int[] count = new int[10];

		for (int i = 0; i < n; i++) {
			count[(nums[i]/exponent) % 10]++;
		}

		for (int i = 1; i < 10; i++) {
			count[i] += count[i-1];
		}

		for (int i = n - 1; i >= 0; i--) {
			output[count[ (nums[i]/exponent) % 10] - 1] = nums[i];
			count[(nums[i]/exponent) % 10]--;
		}

		for (int i = 0; i < n; i++) {
			nums[i] = output[i];
		}
	}

	public static void radixSort(int[] nums) {
		int max = getMax(nums);
		for (int exponent = 1; max / exponent > 0; exponent *= 10) {
			countSort(nums, exponent);
		}
	}
}

File: program_data/PKU_raw/75/63.c
int main()
{
	int n = 0, i, j, x[1000], y[1000], count[1001] = {0}, max = 0;
	while (1)
	{
		cin >> x[n++];
		if (cin.get() == '\n') break;
	}
	for (i = 0; i < n; i++)
	{
		cin >> y[i];
		for (j = x[i]; j < y[i]; j++)
		{
			count[j] ++;  // ?????????????
		}
		if (cin.get() == '\n') break;
	}
	for (i = 0; i < 1001; i++)
	{
		if(max < count[i]) max = count[i];
	}
	cout << n << " " << max << endl;
	return 0;
}